pkgs/development/python-modules/pygltflib/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, fetchFromGitHub
+, pythonOlder
+, setuptools
+, dataclasses-json
+, deprecated
+, pytestCheckHook
+}:
+
+let
+  gltf-sample-models = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "glTF-Sample-Models";
+    rev = "d7a3cc8e51d7c573771ae77a57f16b0662a905c6";
+    hash = "sha256-TxSg1O6eIiaKagcZUoWZ5Iw/tBKvQIoepRFp3MdVlyI=";
+  };
+in
+
+buildPythonPackage rec {
+  pname = "pygltflib";
+  version = "1.16.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitLab {
+    owner = "dodgyville";
+    repo = "pygltflib";
+    rev = "da1c687f5ea88d6063616857d54d195fa0739b37";  # no tags in repo, only on PyPI
```

Comment:
Let's open an upstream issue about that.

Reply:
https://gitlab.com/dodgyville/pygltflib/-/issues/66